data/json/snippets/dimensional_exploration.json

```diff
@@ -5,7 +5,47 @@
-        "text": "\nIMMERSION LOG: 27A\n\nINCIDENT REPORT:\n\nSUMMARY:\nUnforeseen EM interference source forced the team to remain in-phase between prime and sub-scape 27.  Expedition members reported observing a spheroid-entity ('PE-01') moving beneath the typically opaque t-substrate.  Interference readings and expedition testimony implies the PE-01 was a potent EM radiation source, and the origin of the interference.  After 30 min, PE-01 receded and the expedition was aborted in accordance of procedure EMER-12.  The expedition team suffered no injuries, physical or mental, during the incident.\n\nIt's become clear we have greatly underestimated the risk posed by phase immersions.  Engineering has been tasked with designing a new, improved protection suite, and has shown promising advances.  Updated emergency procedures are in-progress.\n\nRefer to file EXO-I-271 for full report."
+        "text": "\nIMMERSION LOG: 27A\n\nINCIDENT REPORT:\n\nSUMMARY:\nUnforeseen EM interference source forced the team to remain in-phase between prime and sub-scape 27.  Expedition members reported observing a spheroid-entity ('PE-01') moving beneath the typically opaque t-substrate.  Interference readings and expedition testimony implies the PE-01 was a potent EM radiation source, and the origin of the interference.  After 30 min, PE-01 receded and the expedition was aborted in accordance of procedure EMER-12.  The expedition team suffered no injuries during the incident.\n\nIt's become clear we have greatly underestimated the risk posed by phase immersions.  Engineering has been tasked with designing a new, improved protection suite, and has shown promising advances.  Updated emergency procedures are in-progress.\n\nRefer to file EXO-I-271 for full report."
+      }
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "artifact_scp",
+    "text": [
+      {
+        "id": "altered_phone_scp",
+        "text": "\nDESIGNATION AI-012-PA\n'Bakelite Phone'\n\nCONTAINMENT PROCEDURE:\nThe object should be kept within an anechoic chamber.  Chamber should be actively monitored in case of ████████ emission leakage.\n\nDESCRIPTION/PARATRONICS:\nAn Ericssen ED 702 bakelite phone, red variant. The original rotary dial has been replaced by a bakelite copy of a Yage Y103d spherical doorknob.\nThe phone remains fully operational, and even when unconnected to a functional landline, █████████████████████ █████████████████████████████████ ████████████.\nOrganic matter reacts adversely to such emissions, and the following effects have been observed during live tests: disorientation and fugue states, ████████████████████████████████, disintegration, ████████████████ followed by death.\n\nBACKGROUND:\nRecovered during immersion ███ within a manifested copy of ███ █████ office.  The object appearance and paratronic effects are believed to be an imprint of the popular perception surrounding the ██████████/██████ ███████, even although such ███████ never existed in this shape."
```

Comment:
```suggestion
        "text": "\nDESIGNATION AI-012-PA\n'Bakelite Phone'\n\nCONTAINMENT PROCEDURE:\nThe object should be kept within an anechoic chamber.  Chamber should be actively monitored in case of ████████ emission leakage.\n\nDESCRIPTION/PARATRONICS:\nAn Ericssen ED 702 bakelite phone, red variant.  The original rotary dial has been replaced by a bakelite copy of a Yage Y103d spherical doorknob.\nThe phone remains fully operational, and even when unconnected to a functional landline, █████████████████████ █████████████████████████████████ ████████████.\nOrganic matter reacts adversely to such emissions, and the following effects have been observed during live tests: disorientation and fugue states, ████████████████████████████████, disintegration, ████████████████ followed by death.\n\nBACKGROUND:\nRecovered during immersion ███ within a manifested copy of ███ █████ office.  The object appearance and paratronic effects are believed to be an imprint of the popular perception surrounding the ██████████/██████ ███████, even although such ███████ never existed in this shape."
```